class_generator/class_generator.py

```diff
@@ -331,37 +322,60 @@ def get_arg_params(
     debug_content: Optional[Dict[str, str]] = None,
     add_tests: bool = False,
 ) -> Dict[str, Any]:
+    python_type_mapping: Dict[str, str] = {
+        "int": "int",
+        "dict": "Dict[str, Any]",
+        "str": "str",
+        "bool": "bool",
+        "list_any": "List[Any]",
+        "list_str": "List[str]",
+    }
+    # All fields must be set with Optional since resource can have yaml_file to cover all args.
+    type_mapping: Dict[str, Dict[str, str]] = {
```

Comment:
we should do 
```
if type_ == "<integer>":
    type_for_doc =  "int",
    type_for_init =  f"Optional[int] = None",
```
and so on...
No need any of the dicts

Reply:
but we have duplications; e.g see `dict`

Reply:
duplicate how?

Reply:
you mean the word `int` for example?

then do 

```
if type_ == "<integer>":
    type_for_doc =  "int",
    type_for_init =  f"Optional[{type_for_doc}] = None",
```